ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -368,6 +368,77 @@ objects:
         /bin/openshift-install --dir=/tmp/artifacts/installer create cluster &
         wait "$!"
 
+    # Stream "journalctl -u bootkube" and "kubectl get all -a --all-namespaces -w" into artifact
+    - name: stream-bootstrapping
+      image: ${IMAGE_INSTALLER}
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: cluster-profile
+        mountPath: /etc/openshift-installer
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        trap 'kill $(jobs -p); exit 0' TERM
+
+        mkdir -p .ssh
+        chmod 700 .ssh
+        cp /etc/openshift-installer/ssh-privatekey .ssh/id_rsa
+        cp /etc/openshift-installer/ssh-publickey .ssh/id_rsa.pub
+
+        function stream-bootkube () {
+          ip=${1}
+          while true; do
+            ssh -o "StrictHostKeyChecking=no" core@${ip} sudo journalctl -u bootkube.service -f --no-tail 2>&1
+            echo "=================== journalctl terminated ==================="
+            date
```

Comment:
Can we embed this in the termination marker?  It feels like it might be associated with the next iteration's logs if it comes after a big banner.  Something like:

```sh
echo "========== journalctl terminated $(date --iso=s --utc) =========="
```

would do it.